sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCSuite.scala

```diff
@@ -1103,6 +1103,17 @@ class JDBCSuite extends SparkFunSuite
        """.stripMargin)
 
       val df3 = sql("SELECT * FROM test_sessionInitStatement")
-      assert(df3.collect() === Array(Row(21519, 1234)))
-    }
+      assert(df3.collect() === Array(Row(21519, 1234))
+    )
```

Comment:
This ')' is wrong. Line 1105~1107 from the original have indentation issue.

Reply:
It must have changed when formatting the code using the IDE. Scalastyle checks passed though, but let me rollback that anyway.

Reply:
@dongjoon-hyun done! Thank you!

Reply:
Ur, actually, I meant the original Spark code is also wrong in terms of indentation. You can fix the indentation of original line 1105~1107 here. :)

Reply:
@dongjoon-hyun You are right! I misread the parenthesis. I think now is correct. Thank you for the observation :)